frameit/lib/frameit/runner.rb

```diff
@@ -35,11 +35,15 @@ def run(path, color = nil)
             next # we don't care about watches right now
           end
 
-          Helper.show_loading_indicator("Framing screenshot '#{full_path}'")
-
           begin
             screenshot = Screenshot.new(full_path, color)
-            screenshot.frame!
+            editor = screenshot.mac? ? MacEditor.new(screenshot) : Editor.new(screenshot)
```

Comment:
Hm, I overlooked this condition here at first. Can we somehow make this more explicit?

Reply:
Sure, no problem.  Would it help if I just keep the same syntax as before ala:

```
 if screenshot.mac?	
        editor = MacEditor.new(screenshot)
      else	
        editor = Editor.new(screenshot)
      end
```

Reply:
ok, updated!